scripts/prepare_network.py

```diff
@@ -280,8 +330,11 @@ def set_line_nom_max(n, s_nom_max_set=np.inf, p_nom_max_set=np.inf):
 
     for o in opts:
         if "Co2L" in o:
-            m = re.findall("[0-9]*\.?[0-9]+$", o)
```

Comment:
@Emre-Yorat89 you are absolutely right that `m` is used to switch between reading `co2limit` and `co2base` from the config. The first value `co2limit` means emissions threshold which shouldn't be crossed by the model. The second value `co2base` corresponds to some year in the past which is used as a reference state to compare the modelled emissions against. Like we want our power system to have 50% less emissions as compared with 1990 (a base year assumed in [Kyoto agreement](https://unfccc.int/process-and-meetings/transparency-and-reporting/reporting-and-review-under-the-kyoto-protocol/first-commitment-period/kyoto-protocol-base-year-data-for-the-first-commitment-period-of-the-kyoto-protocol) and quite often used until today).

As far as I understand, you are loading accurate values for a base year 1990, while a user still may want to modify them considering different goals for emissions reductions. So, I'd suggest to 
1) leave in use `m` as a user-defined parameter;
2) in case `len(m) > 0` calculate `co2limit` using this reduction coefficient like `co2limit = m * emission_extractor(filename)`

What is your feeling about that?